packages/cosmic-swingset/src/chain-main.js

```diff
@@ -184,6 +189,8 @@ export default async function main(
     'home',
     `${homedir}/.ag-chain-cosmos`,
   );
+  const logFormat = processValue.getFlag('log_format', 'plain');
```

Comment:
This won't help getting the config from the toml file, right?

Should we try to plumb this as a "swingset" config?